backend/server.js

```diff
@@ -0,0 +1,34 @@
+require("dotenv").config();
+const express = require("express");
+const app = express();
+const jwt = require("jsonwebtoken");
+app.use(express.json);
+
+const posts = [
+  {
+    username: "Kyle",
+    title: "Post 1",
+  },
+  {
+    username: "Jim",
+    title: "Post 2",
+  },
```

Comment:
how come posts is hardcoded here?

Reply:
The reason i did this was to test to make sure that the user authentication was working. I would check by testing the call in the backend through insomina to see if i could grab the posts based on a certain user.

However, since it is for testing i can go and remove it now since i know it works. Thank you for pointing that out.